addons/l10n_eu_oss/tests/test_multivat.py

```diff
@@ -0,0 +1,31 @@
+from .test_oss import OssTemplateTestCase
+from odoo.tests import tagged
+from unittest import SkipTest
+
+@tagged("post_install", "post_install_l10n", "-at_install")
+class TestMultiVatBelgium(OssTemplateTestCase):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref="be"):
+        # check that both Belgian and French localization are installed
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        foreign_tax_template_ref = "fr"
+        foreign_template_vals = cls.env["account.chart.template"]._get_chart_template_mapping()[foreign_tax_template_ref]
+        foreign_template_module = cls.env.ref(f"base.module_{foreign_template_vals['module']}")
+        if foreign_template_module.state != "installed":
+            raise SkipTest(f"Module required for the test is not installed ({foreign_template_module.name})")
+
+
+    def test_create_foreign_tax_from_foreign_vat_fpos(self):
+        foreign_country = self.env.ref("base.fr")
+        self.foreign_vat_fpos = self.env["account.fiscal.position"].create({
+            "name": "FR foreign VAT",
+            "auto_apply": True,
+            "country_id": foreign_country.id,
+            "foreign_vat": "FR23334175221",
+        })
+        self.foreign_vat_fpos.action_create_foreign_taxes()
+        tax = self.env.ref(f"account.{self.env.company.id}_tva_acq_encaissement_super_reduite")
+        self.assertTrue(tax, "a foreign tax should have been instantiated")
+        base_line, tax_line = tax.invoice_repartition_line_ids
+        self.assertNotEqual(base_line.account_id, tax_line.account_id, "The tax line should have been set with another account")
```

Comment:
Discussed with OCO:
Not ok to do it that way as those tests are lost in a module the feature doesn't belong to.

TODO in a further PR: make a full class with several made up countries and templates so we have a common stuff to use in account for those kind of use case that requires several templates.